cime/src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -498,6 +641,33 @@ subroutine prep_glc_merge( l2x_g, fractions_g, x2g_g )
 
   end subroutine prep_glc_merge
 
+
+  subroutine prep_glc_calc_o2x_gx(timer)
+    !---------------------------------------------------------------
+    ! Description
+    ! Create o2x_gx
+
+    ! Arguments
+    character(len=*), intent(in) :: timer
+
+    character(*), parameter :: subname = '(prep_glc_calc_o2x_gx)'
+    ! Local Variables
+    integer eoi
+    type(mct_avect), pointer :: o2x_ox
+
+    call t_drvstartf (trim(timer),barrier=mpicom_CPLID)
+    do eoi = 1,num_inst_ocn
+      o2x_ox => component_get_c2x_cx(ocn(eoi))
+      call seq_map_map(mapper_So2g, o2x_ox, o2x_gx(eoi), &
+                       fldlist='So_blt:So_bls:So_htv:So_stv:So_rhoeff',norm=.true.)
```

Comment:
Actually, this may not be as important or feasible given that these specific fields are used in the new routines below. A compromise would be to introduce a module-level parameter in prep_glc_mod giving this field list, rather than having it duplicated between here and a second place below.

Reply:
I did your first suggestion, because it seemed to make sense to me after looking over that implementation in seq_flds_mod.  Changed in 84e1c872b35a13d9540b507c1d381d08202ba35c.